src/core/scene/a-scene.js

```diff
@@ -56,7 +56,13 @@ module.exports = registerElement('a-scene', {
 
     init: {
       value: function () {
-        this.behaviors = [];
+        this.behaviors = {
+          tick: [],
+          tock: [],
+          tickPostRender: [],
+          tockPostRender: []
+        };
+        this.postNeedsNotify = false;
```

Comment:
Ok I'll take out the dirty flag and emit in add/remove behavior, how does "post-behaviors-changed" sounds?